### DISCUSSION TITLE
Automation of updating submodule and jsons

### DISCUSSION THREAD

**Discussion by tryggvigy** [neherlab/covid19_scenarios#268] [General]
Hey,

I noticed commits like https://github.com/neherlab/covid19_scenarios/commit/3bbd342c6da5b58423870b820bfd2ba61275712f. I think we can automate keeping `covid19_scenarios_data` up to date using a github workflow. I created a draft PR to demonstrate https://github.com/neherlab/covid19_scenarios/pull/267. What are your thoughts? Curious to hear if you think this could be useful @rneher. We could do the same in the `covid19_scenarios_data` repository for any periodic tasks.

**Comment by nnoll** [comment]
I think we solved this by simply integrating the data submodule into this repo. It should reduce this friction completely.
